### PR TITLE
Add org.yacas.yacas

### PR DESCRIPTION
**Pull request by grzegorzmazur**

### CODE REVIEW

File: org.yacas.yacas.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "app-id": "org.yacas.yacas",
+    "branch": "1.6",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.4",
```
Comment (TingPing) [inline org.yacas.yacas.json]: Any reason to not use 1.6?

File: org.yacas.yacas.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "app-id": "org.yacas.yacas",
+    "branch": "1.6",
```
Comment (TingPing) [inline org.yacas.yacas.json]: Remove the "branch", flathub overrides it anyway.

File: org.yacas.yacas.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                {
+                    "type": "archive",
+                    "url": "https://github.com/grzegorzmazur/yacas/archive/v1.6.1.tar.gz",
+                    "sha256": "6b94394f705bed70a9d104967073efd6c23e9eb1a832805c4d805ef875555ae5"
+                }
+            ],
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app",
```
Comment (TingPing) [inline org.yacas.yacas.json]: Install prefix is already set for you.

File: org.yacas.yacas.json (hunk 1 above)
Comment (grzegorzmazur) [inline org.yacas.yacas.json]: Not at all, it just was originally made when 1.4 was the current version  and I must have missed the updates. Thanks!

**Comment by TingPing** [comment]
Needs appdata file created, `"rename-desktop-file": "yacas-gui.desktop"`, and '"rename-icon": "yacas-gui"`.

**Comment by grzegorzmazur** [comment]
This entry is for yacas, which, contrary to yacas-gui, is a terminal application. Does it still need a .desktop file and icon?

**Comment by TingPing** [comment]
Hmm, well terminal applications can still have a desktop file but honestly I don't think we want cli applications in flathub at the moment.

**Comment by TingPing** [comment]
Sorry about that, I made an extra note about it here: https://github.com/flathub/flathub/wiki/Review-Guidelines#desktop-integration

**Pull request closed without merging**